languages/csharp/exercises/concept/nullability/.meta/Example.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+public static class Badge
+{
+    public static string PrintLabel(string label, int? maximumWidth)
+    {
+        if (maximumWidth == null)
+        {
+            return label;
+        }
+        else
+        {
+            var output = "";
+            for(int i=0;i<label.Length;i+=maximumWidth.Value)
+            {
+                output += label.Substring(i, Math.Min(maximumWidth.Value, label.Length - i)) + "\n";
+            }
+            return output.Trim();
+        }
+    }
+
+    public static string Label(int id, string name, string? department)
+    {
+        var separator = " - ";
+        return $"[{id}]{separator}{name}{department?.ToUpper()?.Insert(0, separator) ?? separator + "GUEST"}";
```

Comment:
I think you can simplify this to:

```suggestion
        return $"[{id}]{separator}{name}{separator}{department?.ToUpper() ?? "GUEST"}";
```

Reply:
Oh yes :) the spec changed :)

Reply:
Yep. I hate it when that happens IRL :D